packages/govern-contract-utils/test/erc1167-proxy-factory.test.ts

```diff
@@ -0,0 +1,120 @@
+import { expect } from 'chai'
+import { ethers } from 'hardhat'
+import { Signer } from 'ethers'
+import { defaultAbiCoder } from 'ethers/lib/utils'
+import { generateCodeFromContract } from './helpers'
+import {
+  CloneFactoryMockFactory,
+  CloneFactoryMock,
+  ClonedContract,
+  ClonedContractWithInit
+} from '../typechain'
+import
+  * as ClonedContractArtifact
```

Comment:
Maybe instead of importing the contract artifacts this way to get the abi. We can use the contract names, I think `ethers.getContractAt` accept the contract name as well.

Reply:
Thanks for the hint.. will check that closer 👍